### DISCUSSION TITLE
API in Docker

### DISCUSSION THREAD

**Discussion by Sensiblemnd** [budlib/budlib-api#16] [Ideas]
Anyway to have the API in Docker? That way anyone that is not comfortable setting up an API using Maven Etc can use it more easily and just work on Front end?

**Comment by Sensiblemnd** [comment]
I would love to help in the front end but not comfortable with the API

**Reply by zbhavyai** [reply]
Thanks for showing your interest. For running the API, you need to have MySQL running (can be possibly in a docker) and run the JAR file. When you use the JAR file, you won't need to use maven.

As of now, we haven't explored docker, but will definitely look into it, and possibly add some related steps for setting up in docker.

**Reply by mikeePy** [reply]
> I would love to help in the front end but not comfortable with the API

Hi Sensiblemnd,

Thank you for taking interest in the BudLib project! :)

We would be happy to have you onboard!

As Bhavyai mentioned, we haven't explore with having the API within Docker, but will definitely be looking into it.

**Reply by zbhavyai** [reply]
@Sensiblemnd I created a wiki page that should answer your question

You can access the wiki here: [https://github.com/budlib/budlib-api/wiki/Deployment-on-Docker-containers](https://github.com/budlib/budlib-api/wiki/Deployment-on-Docker-containers)
